python/lsst/validate/base/output.py

```diff
@@ -0,0 +1,18 @@
+# See COPYRIGHT file at the top of the source tree.
+from __future__ import absolute_import, division, print_function
+
+import json
+
+from .measurement import MeasurementSet, Measurement
+
+
+def output_measurements(package_name, measurement_dict):
```

Comment:
Ok, cool. Especially since we're merging to an in-progress epic branch, we can sort out details as other parts are finished.